pandas/tests/reductions/test_reductions.py

```diff
@@ -1534,3 +1534,30 @@ def test_multimode_complex(self, array, expected, dtype):
         # Complex numbers are sorted by their magnitude
         result = Series(array, dtype=dtype).mode()
         tm.assert_series_equal(result, expected)
+
+
+@pytest.mark.parametrize("dtype", ["float32", "float64"])
```

Comment:
I think instead of these tests, might be better to test in `pandas/test/test_nanops` that bottleneck is not used i.e. test `_bn_ok_dtype` doesn't allow mean.

IMO these tests are essentially testing numpy behavior which ideally numpy should testing the precision instead.

Reply:
Fair point, I guess it's quite safe to trust numpy have a reasonably precise implementation of `sum`. I'll update